src/views/ManageList.jsx

```diff
@@ -1,7 +1,80 @@
-export function ManageList() {
+import { addItem } from '../api/firebase';
+import { useRef } from 'react';
+
+export function ManageList({ listPath }) {
+	const formRef = useRef();
```

Comment:
the best practice recommendation by the React team is to declare this as `const formRef = useRef(null)` if you are going to assign the reference to a DOM element. I'm honestly not sure the different `null` vs `undefined` makes in the initial declaration in reality, but I would still recommend doing it so that your code can be as consistent as possible.

Reply:
Noted. I will add this change. Thanks!